debian/changelog

```diff
@@ -1,3 +1,11 @@
+citus (8.0.3.citus-1) stable; urgency=low
+
+  * Fixes maintenance daemon panic due to unreleased spinlock
+
+  * Fixes an issue with having clause when used with complex joins
+
+ -- Hanefi Onaldi <hanefi@citusdata.com>  Wed, 9 Jan 2019 9:50:07 +0000
```

Comment:
@velioglu Automation script wrote `9:50:7` as the timestamp - missing a leading zero. I fixed it manually.

FYI, I am planning to commit my fixes to small problems such as this.